src/template-tags/tickets.php

```diff
@@ -644,7 +644,7 @@ function tribe_tickets_get_template_part( $slug, $name = null, array $data = nul
 
 	if ( ! $echo ) {
 		// Return should come at the end
-		return $html;
+		return isset( $html ) ? $html : '';
```

Comment:
Hi @stian-overasen,

Thanks for contributing with this, would you mind making a change on the code here, to improve reliability moving forward?

Just after the function starting here we should have `$html = null;` which would prevent this notice in general.

Reply:
I agree. That would be better. Updated now.